public/spec/featureSpec.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+describe("Feature Test", function() {
+  var game, frame, strikeFrame, spareFrame;
+  beforeEach(function() {
+    game = new Game();
```

Comment:
think about how you'd link this file to its source file, for jasmine CLI integration...